src/index.spec.ts

```diff
@@ -1,7 +1,20 @@
-import index from "./index";
+import { mockProcessStdout } from "jest-mock-process";
+
+import start from "./index";
 
 describe("Index", () => {
-  it("returns true", () => {
-    expect(index()).toEqual(true);
+  describe("CLI", () => {
+    let stdoutMock: jest.SpyInstance<boolean>;
+
+    beforeEach(() => {
+      stdoutMock = mockProcessStdout();
+
+      start();
+    });
+
+    it("asks for the world size", async () =>
+      expect(stdoutMock).toHaveBeenCalledWith(
+        'Enter space-delimited world size (e.g. "3 5"): '
+      ));
```

Comment:
I would have very much liked to write a full set of end-to-end integrations tests for the CLI itself but it's a non-trivial task mocking standard IO so it was omitted from the PoC. To be raised for open discussion with the development team.